### DISCUSSION TITLE
CD9000

### DISCUSSION THREAD

**Discussion by KC5VDJ** [kej715/DtCyber#92] [General]
What was this used for?  Stay away from irix 5.3!

Some sort of server appliance?  An indigo 2 would have been a better ICEM option, so I'm guessing server appliance / x workstation terminal for VE?

They did though include the camera, and audio in/out, at least the ones that weren't OEM.

On a side note, at what scale was the DG OEM "Cyber" 120 out there?  I take it that C120 mode (Viking 721) equates to "small cyber" in the rom doc?

**Comment by kej715** [comment]
CD9000 were ICEM graphics workstations. Cyber 120, IIRC, were DG Nova or Eclipse mini's mainly used as batch stations.

**Reply by KC5VDJ** [reply]
Interesting.  I knew about SGI OEMing out, the "Tandem" SMCPs for one, but until reading about the 120, I was unaware of DG doing it.

So freaking glad I came in second for that SGI admin job at Honeywell / Bendix around the turn of the century.  Worked at DFAS in the same building for a while (KCMO), scary enough.  They found BeO in the ventilation all the way to GSA and IRS way at the other end.  The whole building is toxic.  Bannister Federal Complex.  Honeywell, previously Bendix made triggers there, and BeO reflectors are kinda necessary, for AEC then DoE.  At the time it was indys, indigo 1 & 2, o2, etc... Feel sorry for the guy who got the job...at BeO central.  What the hell are they gonna do with that building?  As far as I know at last check, it was simply abandoned, and it is huge, aerials don't give it justice "this hallway is 4800 feet long", jarheads doing their PFT completely inside (3 miles), etc. This isn't asbestos...and the area is densely populated.  Talk about Superfund issues...cleaning Bannister could break the Superfund.

**Reply by KC5VDJ** [reply]
Come to think of it, although the indigo 2 could do ICEM type tasks far better, they were kinda pricey iirc...indys were cheap, competitive with low-end sparc of the day, (I don't recall any usable PA in that price range) down to well-equipped PC prices iirc.  They made great appliances too.  As a cruncher, it was a very basic workstation tho, but had above average video in the price class, plus gl, as well as two scsi-uw, and single ended 50-pin, and iirc, a differential option.  VERY competitive with the high end pc option back then.

Always got a kick out of how "Hackers" (1994 or 1995) ripped their visual file manager for the ever so laughable mainframe.  SGI clearly has prior art.  Did ayone ever use that thing more than once or twice? 🤣 (Neat concept, but gets old quickly as a file manager.  Honestly, the only one worth a damn from my experience is Directory Opus, derived from Amiga's Opus, derived from probably Norton Commander. Works great, the only one I have ever found worth paying for, configurable as hell, and not just cosmetics, which are nice too, and also configurable to a very large degree, not meaning to advertise, just saying.  Came across Directory Opus in a Humble Bundle, they get some gems).

**Reply by wschaub** [reply]
The R5000 MIPS equipped indys with enough RAM in them were pretty snappy under 6.5.11. I used to have one on my desktop alongside a Tadpole SPARCbook 3GX. still have both, not sure if the indy still runs but the sparcbook still does.

**Reply by KC5VDJ** [reply]
Yeah, once irix 6 came out, a lot of bugs were fixed.  5.3 was a nightmare for our shop at the time, VERY creative workarounds for race conditions.  Gotta love deeply nested inline / rvalue conditionals (?:), so useful.  Speaking of which, I need to mod an old hex/octal/decimal/binary/ASCII dumper of mine to do cyber too.  One fprintf / sprintf to rule them all 😎 (the format too)!

**Comment by nickglazzard1959** [comment]
I have mostly fond memories of IRIX and SGI workstations, starting with the
3000 series (68020 based IIRC). Have some boards from one framed on the
wall here. Donated an Octane and Indigo 2 to TNMOC when I left the U.K.
Other Unix systems in the '80s ... not so fond memories ... :-)

**Reply by KC5VDJ** [reply]
Seeing the prices on eBay, getting one again would be tempting given enough cash on hand.  Actually not that bad in price.
